src/CollapsableMixin.js

```diff
@@ -1,103 +1,133 @@
 var React = require('react');
 var TransitionEvents = require('./utils/TransitionEvents');
```

Comment:
Or would this work: http://facebook.github.io/react/docs/animation.html#low-level-api-reacttransitiongroup